src/main/java/in/projecteka/consentmanager/clients/model/PatientLinkReferenceResult.java

```diff
@@ -0,0 +1,38 @@
+package in.projecteka.consentmanager.clients.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import in.projecteka.consentmanager.link.discovery.model.patient.response.GatewayResponse;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+import java.util.UUID;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(Include.NON_NULL)
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Data
+public class PatientLinkReferenceResult {
+    private UUID requestId;
+    private String timestamp;
+    private Link link;
+    private Error error;
```

Comment:
Try to follow similar convention everywhere, In some places we have suffix response.

Check with people please.

Reply:
I think once we start deprecating and removing the older codes, we can reuse some of the names. Right now, we are struggling with names :)